.github/workflows/publish-docker-helm.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           echo "DOCKER_IMAGE_NAME=$(echo $DOCKER_IMAGE_NAME)" >> $GITHUB_ENV
           echo "DOCKER_IMAGE_NAME_TAG=$(echo $DOCKER_IMAGE_NAME_TAG)" >> $GITHUB_ENV
       - name: Build docker image
-        run: mvn install -B -pl :${{ github.event.inputs.app-name }}-module,:${{ github.event.inputs.app-name }}-app,:${{ github.event.inputs.app-name }}-app-docker -P docker
+        run: mvn install -B -pl :${{ github.event.inputs.app-name }}-app-module,:${{ github.event.inputs.app-name }}-app,:${{ github.event.inputs.app-name }}-app-docker -P docker
```

Comment:
Being dependent on format of artifact names is not a good idea. We don't follow this format strictly and it is not documented anywhere.
Even two `lighty-applications` we have use different format. Module in RNC is called just lighty-rnc-**module** while in RCgNMI it's called lighty-rcgnmi-**app-module**. So this postfix you added will probably not work for RNC.